### PR TITLE
Reduce PR template noise

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
This reduces the PR template size.  When you first open a PR, the text box is 10 lines tall - I always find myself just deleting all those comments so I can answer the actual sections of the PR.  With this the important things we want to capture are front and centre.

Changes:
* First 10 lines are the important headings we want every PR to include
* Removed questionnaire comment (don't think anyone was answering this anyway, I think we will get more value from continuing to encourage people to come talk to us)
* Replaced lengthy comments regarding AB tests and AMP with links to docs and moved to the bottom
* Added some docs for AMP, because they didn't exist and I needed something to link to (could do with improvements still I'm sure)

## What is the value of this and can you measure success?
We have some docs on AMP, particularly on validation.  PR template is cleaner and simpler.

## Does this affect other platforms - Amp, Apps, etc?
No

## Request for comment
@guardian/dotcom-platform

### CODE REVIEW

File: docs/03-dev-howtos/17-working-with-amp.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* Check out the existing AMP work in frontend (in IntelliJ: `shift` `shift` `amp`)
+* Get to know the [tools](#tools)
+
+## <a name="tools"></a>Tools & Hints
+
+### Running AMP
+AMP content can be reached in various ways:
+* **Production** - `amp.theguardian.com/<path>`
+* **Code & local** - append an `amp` parameter to your query string e.g. `localhost:9000/an-article?amp`
+* **Beta** - `beta.amp.theguardian.com/<path>`
```
Comment (gtrufitt) [inline docs/03-dev-howtos/17-working-with-amp.md]: I did not know this existed :thinking:

**Review by gtrufitt — APPROVED** [review]
👍 Great!

File: docs/03-dev-howtos/17-working-with-amp.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    * Run these like any other scala test using `sbt` - e.g. `test-only *ArticleTestSuite`
+    * If adding a new endpoint to the tests, make sure you run the test and commit the auto-generated test fixture - you should find a new file generated with a hash filename.
+    * We do not use a live version of the validator, so if you find a discrepancy between the results of these tests and another validation method, this may be the reason.  All you need to do to update the validator itself is delete the file under `data/amp-validator`, re-run the tests and commit the new validator fixture.
+* [Guardian amp validation tool](https://github.com/guardian/frontend/tree/master/tools/amp-validation)
+    * 3 ways this tool can be used:
+        1. Validate your local changes on a hardcoded set of endpoints - `make validate-amp`
+        2. Validate `amp.theguardian.com` on the same endpoints - `node ./tools/amp-validation/index.js`
+        3. Validate `amp.theguardian.com` on today's top 50 most-read articles (according to Ophan) - `node ./tools/amp-validation/top-traffic.js`
```
Comment (kelvin-chappell) [inline docs/03-dev-howtos/17-working-with-amp.md]: This validation section is very useful!

File: docs/03-dev-howtos/17-working-with-amp.md (hunk 1 above)
Comment (jfsoul) [inline docs/03-dev-howtos/17-working-with-amp.md]: It's how to reach the amp content through fastly beta - so points to PROD, same as `beta.theguardian.com`.

Comment (kelvin-chappell) [inline docs/03-dev-howtos/17-working-with-amp.md]: Got it.

**Pull request merged**